src/jit/jit-irgen.c

```diff
@@ -3433,6 +3433,18 @@ static void irgen_op_cover_toggle(jit_irgen_t *g, int op)
    macro_exit(g, JIT_EXIT_COVER_TOGGLE);
 }
 
+static void irgen_op_cover_state(jit_irgen_t *g, int op)
+{
+   jit_value_t shared = irgen_get_arg(g, op, 0);
+
+   uint32_t tag = vcode_get_tag(op);
+   jit_value_t mem = jit_addr_from_cover_tag(tag);
```

Comment:
Please change this to pass the tag number rather than a pointer to the memory (see 140d8d648e). The memory might get reallocated later before the callback fires.